### PR TITLE
RR-132 - Additional tests

### PR DESCRIPTION
**Pull request by mattwills-moj-digital**
Some additional unit and integration tests for the Support Needs tab.

### CODE REVIEW

File: server/views/pages/overview/partials/supportNeedsTabContents.test.ts [hunk 1]
```diff
@@ -22,6 +23,19 @@ describe('Support Needs tab view', () => {
...
+    // Given
+    const supportNeeds = aValidPrisonerSupportNeeds()
+    viewContext = { prisonerSummary, tab: 'support-needs', supportNeeds }
+
+    // When
+    const $ = cheerio.load(compiledTemplate.render(viewContext))
+
+    // Then
+    expect($('#health-and-support-needs-summary-card')).not.toBeUndefined()
+    expect($('#neurodiversity-summary-card')).not.toBeUndefined()
```
Comment (mattwills-moj-digital) [inline server/views/pages/overview/partials/supportNeedsTabContents.test.ts:36]: A bit basic and nothing more than the cypress test is asserting but better than nothing (and we get quick feedback if something is broken, compared to the int tests.

I could drill into the data and assert specific values are shown if we like? A bit fiddly but possible.

File: server/views/pages/overview/partials/supportNeedsTabContents.njk [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
 
-      <div class="govuk-summary-card">
+      <div class="govuk-summary-card" id = "health-and-support-needs-summary-card">
```
Comment (chrimesdev) [inline server/views/pages/overview/partials/supportNeedsTabContents.njk:6]: Minor but white space around the `=`

`id="health-and-support-needs-summary-card"`

Comment (mattwills-moj-digital) [reply server/views/pages/overview/partials/supportNeedsTabContents.njk]: Thanks, I've applied the formatter.

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**